python/requirements.txt

```diff
@@ -40,7 +40,6 @@ colorful
 pyyaml
 rich
 gpustat>=1.0.0
-urllib3
```

Comment:
there is still a urllib3 in `requrements-doc.txt` ?

Reply:
That's are for developers for the docs build, it won't get installed by `pip install "ray[all]"`.  It's possible that it can also be removed